suite/tests/linux/syscall_pwait.c

```diff
@@ -50,19 +50,34 @@
 #include <time.h>
 #include <poll.h>
 
+#define MAX_SIG 32
```

Comment:
You're deliberately not checking the real-time signals?  Seems best to check them all.

Reply:
Ok